client/app/components/proptypes.js

```diff
@@ -93,10 +93,45 @@ export const Destination = PropTypes.shape({
   type: PropTypes.string.isRequired,
 });
 
+export const Query = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  description: PropTypes.string,
+  data_source_id: PropTypes.number.isRequired,
+  created_at: PropTypes.string.isRequired,
+  updated_at: PropTypes.string,
+  user: UserProfile,
```

Comment:
You think we could just revert it back to snake case?

Reply:
Yes, it's way better to have it unified

Reply:
It's trickier than I hoped. Not sure how to get around the linter without conversion..
Since I'm using it only for `user.id` in this Alert feature, it doesn't matter. Ok to leave as is?

Reply:
No problem, I also thought it would be easier :+1: